### DISCUSSION TITLE
New pipe error

### DISCUSSION THREAD

**Discussion by andro-pyf** [TeamNewPipe/NewPipe#11333] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

com.google.android.exoplayer2.ExoPlaybackException: Source error
	at com.google.android.exoplayer2.ExoPlayerImplInternal.handleIoException(ExoPlayerImplInternal.java:644)
	at com.google.android.exoplayer2.ExoPlayerImplInternal.handleMessage(ExoPlayerImplInternal.java:616)
	at android.os.Handler.dispatchMessage(Handler.java:98)
	at android.os.Looper.loop(Looper.java:154)
	at android.os.HandlerThread.run(HandlerThread.java:61)

### Additional information

New pipe errori

**Comment by wb9688** [comment, marked as answer]
Discussions are not the place to report bugs and this description is not enough information, which is why we have an issue template to help you tell us all the necessary details. You can report a bug through https://github.com/TeamNewPipe/NewPipe/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml, though please also first read https://github.com/TeamNewPipe/NewPipe/discussions/11260 about the current situation, which might already answer the questions you have and note that v0.27.1 has been released now, which fixes some of the issues.
